cmd/frontend/graphqlbackend/product_subscription_status.go

```diff
@@ -75,3 +75,13 @@ func (productSubscriptionStatus) MaximumAllowedUserCount(ctx context.Context) (*
 func (r productSubscriptionStatus) License() (*ProductLicenseInfo, error) {
 	return GetConfiguredProductLicenseInfo()
 }
+
+func isFreePlan(info *ProductLicenseInfo) bool {
+	for _, tag := range info.TagsValue {
+		if tag == "plan:free-0" {
```

Comment:
```suggestion
		if tag == fmt.Sprintf("plan:%s", licensing.PlanFree0) {
```

I'm scared of this string being missed 😬

Reply:
Would love to do that but `licensing` is an `internal` module. Can maybe do some other form of shuffling